### PR TITLE
Improve tag semantic usage

### PR DESCRIPTION
**Pull request by alfsb**
This commit changes tag usage without changing text contents. I'm in doubt to submit it with or without `[skip-revcheck]`.

### CODE REVIEW

File: language/types/declarations.xml [hunk 1]
```diff
@@ -179,8 +179,8 @@ Stack trace:
      because such a function is contradictory.
      Previously, it already emitted the following
      <constant>E_NOTICE</constant> when called:
-     <literal>Only variable references should be returned by reference</literal>.
-
+     <emphasis role="strong">Only variable references should be returned by
+     reference</emphasis>.
```
Comment (Girgias) [inline language/types/declarations.xml:183]: This ``literal`` tag is to wrap the warning output so should be marked as such. The [``computeroutput``](https://tdg.docbook.org/tdg/5.1/computeroutput) tag might be better and more semantic.

File: reference/mysqli/mysqli/real-escape-string.xml [hunk 2]
```diff
@@ -49,8 +49,9 @@
        The string to be escaped.
       </para>
       <para>
-       Characters encoded are <literal>NUL (ASCII 0), \n, \r, \, ', ", and
-       Control-Z</literal>.
+       Characters encoded are <literal>NUL (ASCII 0)</literal>,
+       <literal>\n</literal>, <literal>\r</literal>, <literal>\</literal>,
+       <literal>'</literal>, <literal>"</literal> and <literal>Control-Z</literal>.
```
Comment (Girgias) [inline reference/mysqli/mysqli/real-escape-string.xml:54]:
```suggestion
       <literal>'</literal>, <literal>"</literal>, and
       <keycombo action='simul'><keycap>CTRL</keycap><keycap>Z</keycap></keycombo>.
```
Oxford comma and see #2387

Comment (Girgias) [inline reference/mysqli/mysqli/real-escape-string.xml:54]:
```suggestion
       <literal>'</literal>, <literal>"</literal> and
       <keycombo action='simul'><keycap>CTRL</keycap><keycap>Z</keycap></keycombo>.
```
See #2387

**Review by Girgias — COMMENTED** [review]
This should be flagged by revcheck as it is a markup change

File: language/types/declarations.xml (hunk 1 above)
Comment (alfsb) [reply language/types/declarations.xml]: Changed.

File: reference/mysqli/mysqli/real-escape-string.xml (hunk 2 above)
Comment (alfsb) [reply reference/mysqli/mysqli/real-escape-string.xml]: Changed.

**Review by Girgias — APPROVED** [review]

**Pull request merged**